axmolengine/axmol
# Release 2.1.5

## Describe your changes

- Use coroutine to control AutoTest flow
- Fix some issues for cpp-tests
- Add otf to android gralde `noCompress`
- Update copyright notice in some sources
- Fix use unexpected libpng from `FindPNG.cmake`
- Improve build system
- Fix checkReallySupportsASTC does not work on ios device
- Update DrawNodeEx to v0.95.1
- Improve FileUtils


## Issue ticket number and link


## Checklist before requesting a review
### For each PR
- [ ] Add Copyright if it missed:   
      - `"Copyright (c) 2019-present Axmol Engine contributors (see AUTHORS.md)."`
       
   Optional:
   - [ ] I have checked readme and add important infos to this PR.
   - [ ] I have added/adapted some tests too.
          
### For core/new feature PR
- [ ] I have checked readme and add important infos to this PR.
